src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/StageDetail.component.js

```diff
@@ -215,6 +215,7 @@ const StageDetailPlain = (props: Props) => {
             return (<Button
                 small
                 secondary
+                icon={<IconAdd24 />}
```

Comment:
Same doubt here about using the `IconAdd16` instead?

Reply:
Thanks, fixed 👍